src/components/CreateForm/textinput.js

```diff
@@ -14,14 +25,30 @@ export default function TextInput({newMeal, setNewMeal}) {
 				required
 			/>
 			<label htmlFor="ingredients">Ingredients:</label>
-			<textarea
-				name="ingredients"
-				id="ingredients"
-				value={newMeal.ingredients}
-				placeholder="1 cucumber, 2 carrots,..."
-				onChange={event => setNewMeal({...newMeal, ingredients: event.target.value})}
-				required
-			/>
+			<StyledInput>
+				<input
+					type="text"
+					name="ingredients"
+					id="ingredients"
+					value={currentIngredient}
+					placeholder="1 cucumber"
+					onChange={event => setCurrentIngredient(event.target.value)}
```

Comment:
It is not really good practice to pass the set function of a state directly as props. This can lead to bugs. What would be better is to pass a callback function which then calls the setter function of the state where the state is. 
```suggestion
					onChange={onCurrentIngredientChange}
```
In your function call as props it would be:
```js
export default function TextInput({
	newMeal,
	setNewMeal,
	currentIngredient,
	onCurrentIngredientChange,
	ingredients,
	setIngredients,
})
```
And then in your form:
```jsx
				<TextInput
					newMeal={newMeal}
					setNewMeal={setNewMeal}
					currentIngredient={currentIngredient}
					onCurrentIngredientChange={(event) => setCurrentIngredient(event.target.value)}
					setIngredients={setIngredients}
					ingredients={ingredients}
				/>
```
The same for `setIngredients`. You always want to update the state in the component where the state actually is :)